src/arch/xtensa/configs/cannonlake_defconfig

```diff
@@ -2,8 +2,8 @@ CONFIG_CANNONLAKE=y
 CONFIG_CAVS_DMIC=y
 CONFIG_CAVS_SSP=y
 CONFIG_CAVS_ALH=y
-CONFIG_CAVS_LPS=y
-CONFIG_CAVS_LPRO=y
+CONFIG_CAVS_LPS=n
+CONFIG_CAVS_LPRO=n
```

Comment:
They are both disabled by default so you don't have to turn them off in defconfig

Reply:
ACK I'll remove duplicating settings.